plugins/kotlin-as-java/src/main/kotlin/signatures/JavaSignatureProvider.kt

```diff
@@ -30,8 +30,9 @@ class JavaSignatureProvider internal constructor(ctcc: CommentsToContentConverte
     private val ignoredVisibilities = setOf(JavaVisibility.Default)
 
     // R3: Hide 'Final' modifier from generated docs.
+    // R3: Remove hiding 'Final' modifier.
```

Comment:
No need to add this a log of changes. Delete both comments.

Reply:
Deleted both comments.